src/components/DropDowns/UserAndUnapprovedTimeEntriesDropDown/UserAndUnapprovedTimeEntriesDropDown.tsx

```diff
@@ -0,0 +1,83 @@
+import { StyleSheet, View } from "react-native";
+
+import {
+  TimeEntry,
+  UserAndUnapprovedTimeEntriesType,
+} from "../../../utilily/types";
+import { MainLabel } from "./MainLabel";
+import { useState } from "react";
+import { InfoRow } from "./InfoRow";
+import colors from "../../../assets/theme/colors";
+import { includes, pull } from "lodash";
+
+const countAllEntries = (
+  usersWithUnconfirmedTimeEntries: UserAndUnapprovedTimeEntriesType[],
+) =>
+  usersWithUnconfirmedTimeEntries.reduce(
+    (total, current) => total + current.unapprovedTimeEntries.length,
+    0,
+  );
+
+type Props = {
+  usersTimeEtries: UserAndUnapprovedTimeEntriesType[];
+};
+export const UserAndUnapprovedTimeEntriesDropDown = ({
+  usersTimeEtries,
+}: Props) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const [onCheck, setOnCheck] = useState<TimeEntry["id"][]>([]);
+  const adminName = `${usersTimeEtries[0].adminFirstName}\u00A0${usersTimeEtries[0].adminLastName}`;
```

Comment:
What is this `\u00A0`?
I've noticed it in a few places.

Reply:
It gives a space between words. I like it because it shows that space there is meant to be, it's not just " ".  I think that " " looks a bit hardcoded.
It is good to know also because \u00A0 is not the same as " ". it gives the visual effect of space but it's not space. I think it can be pretty useful sometimes. 
But I understand that it is harder to read, so probably I won't use it a lot. Maybe only for some functions where I need that it doesn't take it as space.